static/js/message_events.js

```diff
@@ -143,6 +143,7 @@ exports.update_messages = function update_messages(events) {
         const new_topic = util.get_edit_event_topic(event);
 
         const {new_stream_id} = event;
+        const new_stream = stream_data.get_sub_by_id(new_stream_id);
```

Comment:
Maybe add a comment just above this line that we may not have access to the new stream, so this could return undefined, as a reminder to folks reading this code?